.github/workflows/stale.yaml

```diff
@@ -0,0 +1,21 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: This issue has been marked as stale because it has been open for 360 days with no activity. Please remove the stale label or comment or this issue will be closed in 5 days.
+          close-issue-message: This issue was closed because it has been inactive for 365 days with no activity.
+          stale-pr-message: This pull request has been marked as stale because it has been open for 90 days with no activity. Please remove the stale label or comment or this pull request will be closed in 5 days.
+          close-pr-message: This pull request was closed because it has been inactive for 95 days with no activity.
+          days-before-issue-stale: 360
```

Comment:
@chipzoller know roughly how many issues this would cloud? if it's many, should we start with 720 or another number > 360? I do think it's valuable for someone to review closed issues to make sure we aren't accidentally closing important ones. I'm happy to contribute time here.

Reply:
Sg, I'm happy to use that comment as a dry run. That said, eye balling backlog it looks this would be 50-60 issues which feels like will be a fair amount of issues to review at once :)

Reply:
Ok, looks like we're resolved here then. Will keep an eye out and adjust as necessary.